lab_7_llm/assets/styles.css

```diff
@@ -0,0 +1,114 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap');
+
+@font-face {
+    font-family: 'Gilroy';
+    src: local('Gilroy'),
+    url("./font/Gilroy-ExtraBold.woff") format("woff");
+    url("./font/Gilroy-Light.woff") format("woff");
```

Comment:
It seems to me that each font type should be imported separately under its own name, plus there is a syntax error